sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -769,6 +772,11 @@ class MLPClassifier(BaseMultilayerPerceptron, ClassifierMixin):
         (how many times each data point will be used), not the number of
         gradient steps.
 
+    max_fun : int, optional, default 200
```

Comment:
why 200 ? any idea on the expected number of function calls when the number of iterations achieves 200 ?

Reply:
In the test experiments I ran it appeared that num_iters and num_funcalls was about the same (typically num_funcalls = num_iters + 3, or some other small number), so I chose 200 to match the default for max_iters.
However, on second thought this may cause very different behavior since if you set max_iters > 200, it will stop at 200 unless you also change max_fun... so I'll change this default to 15000 to match the default in fmin_l_bfgs__b, so that the default behavior is the same as it was before.
However, if someone (like me) needs more than 15000, they can set it using the max_fun parameter.